VRCSubs/vrcsubs.py

```diff
@@ -183,12 +183,19 @@ def _process_osc(self):
 def main():
     global config
     # Load config
-    cfgfile = f"{os.path.dirname(os.path.realpath(__file__))}/Config.yml"
-    if os.path.exists(cfgfile):
+    cfgfile = f"{os.getcwd()}/config.yml"
```

Comment:
`os.getcwd()` doesn't always return the correct subfolder (such as using the batch scripts to launch), and as a result the VRCSubs config file may get confused with the VRCNowPlaying config file. I think the way it was before was a little better 💛